pkgs/by-name/README.md

```diff
@@ -108,13 +108,13 @@ There's some limitations as to which packages can be defined using this structur
 
 ## Validation
 
-CI performs [certain checks](../test/nixpkgs-check-by-name/README.md#validity-checks) on the `pkgs/by-name` structure.
-This is done using the [`nixpkgs-check-by-name` tool](../test/nixpkgs-check-by-name).
+CI performs [certain checks](https://github.com/NixOS/nixpkgs-check-by-name?tab=readme-ov-file#validity-checks) on the `pkgs/by-name` structure.
```

Comment:
Future: we should publish with GitHub pages so that https://nixpkgs-check-by-name.github.io/ is the URL.

Reply:
Doesn't seem necessary, but yeah that's an option :)